projects/storefrontlib/cms-components/misc/theme-switcher/config/default-theme-switcher-config.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { ThemeSwitcherConfig } from './theme-switcher-config';
+
+export const defaultThemeSwitcherConfig: ThemeSwitcherConfig = {
+  themeSwitcher: {
+    themes: [
+      { i18nNameKey: 'themeSwitcher.themes.default', className: 'santorini' },
+      {
+        i18nNameKey: 'themeSwitcher.themes.highContrastDark',
+        className: 'high-contrast-dark',
+      },
+      {
+        i18nNameKey: 'themeSwitcher.themes.highContrastLight',
+        className: 'high-contrast-light',
+      },
```

Comment:
[QUESTION]

Should the "dark contrast" and "light contrast" be really **altearnatives** to the existing themes (santorini or whatever CMS-driven value set in backoffice)?
Or maybe those "dark contrast" and "light contrast" should be only a modifiers of existing themes?
Just asking, what makes more sense to you?

Reply:
Currently, we should focus on modifying existing themes, as we cannot fully address the customer's changes and their components.